registry/src/main/java/de/unistuttgart/iaas/amyassist/amy/registry/LocationRegistryImpl.java

```diff
@@ -67,7 +67,7 @@ public void save(Location location) {
             boolean hasWork = locs.stream().anyMatch(Location::isWork);
 
             if (hasHome && location.isHome() || hasWork && location.isWork()) {
-                throw new RegistryException("There can only be at most one work and one home entity");
+                throw new RegistryInvalidRequestException("There can only be at most one work and one home entity");
```

Comment:
Why RequestException?

Reply:
You mean that the word `request` is associated with http requests?
`Request` does not refer to an http-request here.

What I mean is that the request to update this registry did not make sense.